ev3dev/core.py

```diff
@@ -44,6 +44,7 @@
 import array
 import mmap
 import ctypes
+import logging
```

Comment:
I think you should put logging into a separate PR. Every import makes startup time a bit longer, so we should only do this if necessary.

Reply:
Fair point, I'll back out the logging parts that are in here and file an issue to chat about what logging we want in core.py.